### PR TITLE
Setup Proyecto

### PR DESCRIPTION
**Pull request by SantiagoFoster**
# Summary
- I added a red "Hello World" webpage.
- I created the .gitignore for the proyect.

# Trello Card
https://trello.com/c/EwlOKjbU/11-setup-del-proyecto

# Screenshots
image

### CODE REVIEW

File: scss/application.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.title {
+	color: red;
```
[the remark this answers is not in the document]
Comment (SantiagoFoster) [reply scss/application.scss]: The training notified me of the convention after I had already pushed this, I'll change it but convention should be fine in the rest of the PRs.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>HTML & CSS Training</title>
+	<meta name="description" content="training">
+	<meta name="author" content="Wolox">
+	<link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+	<h1 class="title">Hello World</h1>
```
[the remark this answers is not in the document]
Comment (SantiagoFoster) [reply index.html]: Do you mean to change the label h1 into div?, or to put this h1 label into a newly created div?

[the remark this answers is not in the document]
Comment (frparodi) [reply index.html]: I mean to put `h1` inside a new `div`. It's not required, but naturally you'll be wrapping all the content into a wrapper :)

File: index.html [hunk 3]
```diff
@@ -9,6 +9,8 @@
 	<link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-	<h1 class="title">Hello World</h1>
+  <div>
+  <h1 class="title">Hello World</h1>
```
Comment (frparodi) [inline index.html]: Add a tab to this line

**Pull request merged**